modules/end-user-guide/pages/creating-a-workspace-by-importing-the-source-code-of-a-project.adoc

```diff
@@ -1,5 +1,5 @@
 [id="creating-a-workspace-by-importing-the-source-code-of-a-project"]
-:navtitle: Creating a workspace by importing the source code of a project
+:navtitle: Creating a workspace with your project's source code
```

Comment:
I would keep `importing` in here. It is possible to rephrase it better, but I would definitively go with `importing` :)